### PR TITLE
Fix #1346: Frontend for sending email to invite users to participant team using URL.

### LINKED ISSUE #1346: Add feature to invite users to participant team using URL
## Deliverables

Add a feature to invite new team member to a participant team using a Unique URL. Also, don't forget to write unit tests. :) 

## Steps

1. **user1** enters the email of his teammate (let's say **user2**)
2. Send an email to **user2** (if he/she is registered on EvalAI) with a unique URL
3. **user2** clicks on the URL to accept the invitation to the team.
4. *user1* gets an email saying that "**user2** has accepted your invite to team <blah>" (we can iterate over the email message later) 

### Some important points to note

- [ ] Checks performed before inviting to the team should be similar to current invitation logic

### PR DESCRIPTION
**Pull request by isht3**
* Define new route for user's unique link.

     * Send $location.absUrl() to backend.

     * User's invite accepted feature.

#1346

### CODE REVIEW

File: frontend/src/js/route-config/route-config.js [hunk 1]
```diff
@@ -127,6 +127,16 @@
             authenticate: true
         };
 
+        var invitation = {
+            name: "web.invitation",
+            parent: "web",
+            url: "/invitation/:hash/:encoding",
```
Comment (aka-jain) [inline frontend/src/js/route-config/route-config.js]: HI @isht3 , please change the name of hash and encoding to something meaningful and non redundant. Actually hash and encoding are very common words.

Comment (isht3) [reply frontend/src/js/route-config/route-config.js]: @aka-jain How about `email_hash` and, `user_enc`?

File: frontend/src/js/controllers/teamsCtrl.js [hunk 2]
```diff
@@ -341,7 +341,8 @@
                 parameters.url = 'participants/participant_team/' + participantTeamId + '/invite';
                 parameters.method = 'POST';
                 parameters.data = {
-                    "email": result
+                    "email": result,
+                    "url": $state.href('web.challenge-host-teams', {}, {absolute: true})
```
Comment (spyshiv) [inline frontend/src/js/controllers/teamsCtrl.js]: @isht3 please change the url here.
Need to replace `web.challenge-host-teams` with `web.teams` 

also you need to change 
`full_url = url + "invitation/" + unique_hash`
to 
`full_url = url + "participant-invitation/" + unique_hash`

[Source](https://github.com/Cloud-CV/EvalAI/pull/1489/files#diff-936afd091c6c99c1a02293abc3be8653R264)

**Comment by isht3** [comment]
@spyshiv Done!

**Comment by spyshiv** [comment]
LGTM :) This PR can be merged after #1414 
Thanks @isht3

**Comment by RishabhJain2018** [comment]
Closing this due to inactivity.

**Comment by RishabhJain2018** [comment]
Closing this due to inactivity.

**Pull request closed without merging**